weavelet.go

```diff
@@ -225,12 +225,12 @@ func (w *weavelet) start() (Instance, error) {
 		startWork(w.ctx, "serve weavelet conn", remote.conn.Serve)
 
 		lis := remote.conn.Listener()
-		addr := call.NetworkAddress(fmt.Sprintf("tcp://%s", lis.Addr().String()))
+		addr := call.NetEndpoint{Net: "tcp", Addr: lis.Addr().String()}
```

Comment:
Maybe change `addr` to `string` instead of `call.NetEndpoint` since we never dial it and only ever call the `String` method on it? Up to you.

Reply:
I like that! Done